spec/spec_helper.rb

```diff
@@ -2,7 +2,7 @@
 require 'bundler/setup'
 require 'consul_application_settings'
 
-Dir[File.expand_path('support/**/*.rb', __dir__)].each { |f| require f }
+Dir[File.expand_path('_support/**/*.rb', __dir__)].each { |f| require f }
```

Comment:
`_support ` What rules are describing to use underscore in a folder name?

Reply:
No rules. I just like to distinguish it from folders with tests.
I can roll it back if you don't like it.

Reply:
I don't like it. support is standard naming across internet for all helpers in spec

Reply:
No problem. Fixed.